darwin/importer/formats/coco.py

```diff
@@ -112,7 +118,10 @@ def parse_annotation(annotation: Dict[str, Any], category_lookup_table: Dict[str
     iscrowd = annotation.get("iscrowd") == 1
 
     if iscrowd:
-        print("Warning, unsupported RLE, skipping")
+        logger.warn(
+            f"Skipping annotation {annotation.get('id')} because it is a crowd"
+            "annotation, and this is not supported from Coco."
+        )
```

Comment:
Unless I'm mistaken, coco format supports iscrowd? Unless we just mean our coco importer/exporter? In which case maybe want to differentiate between coco the format and our code.

Reply:
Yeah, I can clarify this - Coco _does_ support `iscrowd`, _we_ don't import it - in fact that's why this ticket exists.